docs/installation/server-config/index.rst

```diff
@@ -1,172 +1,233 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The maximum number of memory bytes to be used for buffering across all threads. The default value in KSQL is ``10000000`` (~ 10 MB).
+Here is an example to change the value to ``20000000`` by using the KSQL command line:
 
-   .. code:: bash
+.. code:: bash
 
-        ksql> SET 'cache.max.bytes.buffering'='20000000';
+    ksql> SET 'cache.max.bytes.buffering'='20000000';
 
-   For more information, see the :ref:`Streams parameter reference <streams_developer-guide_optional-configs>` and :cp-javadoc:`Javadoc|streams/javadocs/org/apache/kafka/streams/StreamsConfig.html#CACHE_MAX_BYTES_BUFFERING_CONFIG`.
+For more information, see the :ref:`Streams parameter reference <streams_developer-guide_optional-configs>` and :cp-javadoc:`Javadoc|streams/javadocs/org/apache/kafka/streams/StreamsConfig.html#CACHE_MAX_BYTES_BUFFERING_CONFIG`.
 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 fail.on.deserialization.error
-    Indicates whether to fail if corrupt messages are read. KSQL decodes messages at runtime when reading from a Kafka topic. The
-    decoding that KSQL uses depends on what's defined in STREAM's or TABLE's data definition as the data format for the
-    topic. If a message in the topic can't be decoded according to that data format, KSQL considers this message to be
-    corrupt. For example, a message is corrupt if KSQL expects message values to be in JSON format, but they are in
-    DELIMITED format. The default value in KSQL is ``true``. For example, to ignore corrupt messages, add this to your
-    properties file:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Indicates whether to fail if corrupt messages are read. KSQL decodes messages at runtime when reading from a Kafka topic. The
+decoding that KSQL uses depends on what's defined in STREAM's or TABLE's data definition as the data format for the
+topic. If a message in the topic can't be decoded according to that data format, KSQL considers this message to be
+corrupt. For example, a message is corrupt if KSQL expects message values to be in JSON format, but they are in
+DELIMITED format. The default value in KSQL is ``true``. For example, to ignore corrupt messages, add this to your
+properties file:
 
-    .. code:: java
+.. code:: java
 
-        fail.on.deserialization.error=false
+    fail.on.deserialization.error=false
 
+^^^^^^^^^^^^^^^^^^^^^^^^^
 ksql.command.topic.suffix
-    The KSQL server command topic determines the resource pool. By default, KSQL servers use the ``ksql__commands`` command topic.
-    To assign a server to a different pool, change the ``ksql.command.topic.suffix`` setting. For example, if you change to ``'production_commands'``, in the ``ksql-server.properties`` file, the command topic will be named ``ksql__production_commands``.
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The KSQL Server command topic determines the resource pool. By default, KSQL Servers use the ``ksql__commands`` command topic.
```

Comment:
As above, these defaults are now out dated and the `ksql.command.topic.suffix` doesn't exist. See the above comment for the correct content.